run().catch(console.error)

async function run() {
  console.log('Hello from TypeScript!')
}
